File: Exercício_18.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[])
{
    float peso, altura, imc;
    char sexo;

    // Exibe as informações sobre o app na tela

    printf("App para cálculo e validacao do IMC \n ");

    printf("Digite o seu peso em kg: "); // Insercao do peso
    scanf("%f", &peso);

    printf("Digite a sua altura em metros: "); // Insercao da altura
    scanf("%f", &altura);

    printf("Digite o seu sexo : "); // Insercao do sexo
    scanf("%s", &sexo);

    // Calculo do IMC
    imc = peso / (altura * altura);

    if (sexo == 'm' || sexo == 'M')
    {
        if (imc < 20)
        {
            printf("Voce esta abaixo do peso ideal");
        }
        else if (imc >= 20 && imc <= 25)
        {
            printf("Voce esta no peso ideal");
        }
        else if (imc > 25)
        {
            printf("Voce esta com sobrepeso");
        }
    }
    else if (sexo == 'f' || sexo == 'F')
    {
        if (imc < 19)
        {
            printf("Voce esta abaixo do peso ideal");
        }
        else if (imc >= 19 && imc <= 24)
        {
            printf("Voce esta no peso ideal");
        }
        else if (imc > 24)
        {
            printf("Voce esta com sobrepeso");
        }
    }
return 0;
}

File: Exercício_18.py
P = float(input('Digite o peso em Kg: '))
A = float(input('Digite a altura em m: '))
Sexo = str(input('Digite o sexo M ou F: '))

Imc = P/(A*A)

print(f"O IMC é: {Imc:.2f}")


if ((Sexo == 'f') or (Sexo =='F')):
 print(f"Resultado para o sexo feminino:")
elif Imc<19:
 print(f"Abaixo do peso!")
elif Imc>=19 and Imc<24:
 print(f"Peso ideal")
else:
 print(f"Acima do peso!")


if ((Sexo == 'm') or (Sexo =='M')):
 print(f"Resultado para o sexo masculino:")
if Imc<20:
 print(f"Abaixo do peso")
elif Imc>=20 and Imc<25:
 print(f"Peso ideal")
else:
 print(f"Acima do peso!")